### ISSUE TITLE
[Crash] System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service)

### ISSUE THREAD

**Issue by mymande** [sn4k3/UVtools#818]
### System

```Text
UVtools v4.0.6 X64
Operative system: Linux 6.6.6-76060606-generic #202312111032~1702306143~22.04~d28ffec SMP PREEMPT_DYNAMIC Mon D X64
Graphic card: 
Processor: AMD Ryzen 5 6600HS Creator Edition
Processor cores: 12
Memory RAM: 12.06 / 13.34 GB
Runtime: pop.22.04-x64
Framework: .NET 6.0.25
AvaloniaUI: 11.0.6
OpenCV: 4.8.1

Path:       /tmp/.mount_UVtoolsLe41O/usr/bin/
Executable: /tmp/.mount_UVtoolsLe41O/usr/bin/UVtools
```

### Printer and Slicer

- Printer:  Elegoo mars pro
- Slicer:  Lychee slicer 5.4.300

### Description of the bug

When trying to open a sliced file UVTools crashes and I get the following error
```
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service)
 ---> Tmds.DBus.Protocol.DBusException: org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service
   at Tmds.DBus.Protocol.DBusConnection.MyValueTaskSource`1.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at Tmds.DBus.Protocol.DBusConnection.CallMethodAsync(MessageBuffer message)
   at Tmds.DBus.Protocol.Connection.CallMethodAsync(MessageBuffer message)
   --- End of inner exception stack trace ---
Category: Task
File: Hubraft.ctb  [Version: 2] [Class: ChituboxFile]

Machine date time: 1/7/2024 5:39:26 PM
    UTC date time: 1/7/2024 4:39:26 PM
```

### How to reproduce

Open UVTools
File->open->.ctb file

**Comment by sn4k3** [comment]
Duplicate of #803
